lib/pi_piper/driver.rb

```diff
@@ -0,0 +1,114 @@
+module PiPiper
+  class Driver
+    
+    VERSION = '0.1.0'
+    ALLOW = []
+    API_VERSION = '0.1.0'
```

Comment:
I don't think we need to. Since the logic will be inside each gem for each driver, let's say we update BCM. That update (and its version) should not rely on PiPiper version or other driver versions.